fixed deprecation notice about constructor function & check if geshi is already loaded

kminek opened the pull request.

shazahm1 commented:
Rather than modding GeSHi directly, I think it would be best to update the PR to include GeSHi 1.0.8.12

https://github.com/GeSHi/geshi-1.0

kminek commented:
yeah, i'll try to bundle the latest version soon

kminek commented:
ok, i have updated geshi to the latest version (from GitHub) - it works correctly on my blog :) i have added `class_exists` check in order for end-users to swich geshi version easily (for instance some users might use composer which loads different version of geshi)
